docs/src/userguide/loading_iris_cubes.rst

```diff
@@ -51,15 +51,14 @@ The ``air_potential_temperature`` cubes were 4 dimensional with:
 
 .. note::
 
-     The result of :func:`iris.load` is **always** a
-     :class:`list of cubes <iris.cube.CubeList>`.
-     Anything that can be done with a Python :class:`list` can be done
-     with the resultant list of cubes. It is worth noting, however, that
-     there is no inherent order to this
-     :class:`list of cubes <iris.cube.CubeList>`.
-     Because of this, indexing may be inconsistent. A more consistent way to
-     extract a cube is by using the :class:`iris.Constraint` class as
-     described in :ref:`constrained-loading`.
+     The result of :func:`iris.load` is **always** a :class:`iris.cube.CubeList`
+     (even if it only contains one :class:`iris.cube.Cube` - see
+     :ref:`strict-loading`). Anything that can be done with a Python
+     :class:`list` can be done with an :class:`iris.cube.CubeList`.
+     
+     The order of this list should not be relied upon. Ways of extracting a
```

Comment:
```suggestion
     The order of this list should not be relied upon. Ways of loading a
```
I think of "extract" being something you can do when you've already got your cubelist (using its `extract` method) so think I prefer sticking with "load" here.  Feel free to disagree 🙂